README.md

```diff
@@ -57,6 +57,10 @@ For more information on reproducing the benchmarks and running some examples, se
 The support for HF models is provided by our HF model adapter. One can obtain similar latencies as tabulated above with HF models using our HF model adapter:
 
 ```python
+from fms.models import get_model
+from fms.models.hf import to_hf_api
+import torch
+from transformers import pipeline
 # fms model
 llama = get_model("llama", "13b")
```

Comment:
though I don't think this actually gives us a usable model without the `model_path` parameter... enough for a benchmark but I suspect not enough to make the below generator work? @JRosenkranz  does this example need an update to get weights?

Reply:
You are correct, this might not fully work like this.

Reply:
well, I guess we can iterate. I'll just with Josh but for now will merge this part.

Reply:
Ah yes, you would need weights to have it generate real outputs. This should run with no issues, but will not produce a good result.